scripts/launcher/lib/conf/configuration.py

```diff
@@ -55,6 +55,22 @@ def __repr__(self):
         return msg
 
 
+class GlobalConfiguration(BaseConfiguration):
+    """Global static configuration usable from all corners of program"""
```

Comment:
I suggest:
"from all parts of the program"

Reply:
Fixed.